cypress/pageobject/Sample/SampleTestCreate.ts

```diff
@@ -0,0 +1,115 @@
+export class SampleTestPage {
+  visitSampleRequestPage() {
+    cy.verifyAndClickElement("#sample-request-btn", "Request Sample Test");
+  }
+
+  selectSampleType(option: string) {
+    cy.clickAndSelectOption("#sample-type", option);
+  }
+
+  selectIcmrCategory(option: string) {
+    cy.clickAndSelectOption("#icmr-category", option);
+  }
+
+  fillIcmrLabel(label: string) {
+    cy.get("#icmr-label").should("be.visible").type(label);
+  }
+
+  fillFastTrackReason(value: string) {
+    cy.get("#is_fast_track").should("be.visible").check();
+    cy.get("#fast_track").should("be.visible").type(value);
+  }
+
+  fillDoctorName(value: string) {
+    cy.get("#doctor_name").should("be.visible").type(value);
+  }
+
+  fillAtypicalPresentation(value: string) {
+    cy.get("#is_atypical_presentation").should("be.visible").check();
+    cy.get("#atypical_presentation").should("be.visible").type(value);
+  }
+
+  fillDiagnosis(value: string) {
+    cy.get("#diagnosis").should("be.visible").type(value);
+  }
+
+  fillEtiology(value: string) {
+    cy.get("#etiology_identified").should("be.visible").type(value);
+  }
+
+  fillDiffDiagnosis(value: string) {
+    cy.get("#diff_diagnosis").should("be.visible").type(value);
+  }
+
+  checkHasSari() {
+    cy.get("#has_sari").should("be.visible").check();
+  }
+
+  checkHasAri() {
+    cy.get("#has_ari").should("be.visible").check();
+  }
+
+  checkIsUnusualCourse() {
+    cy.get("#is_unusual_course").should("be.visible").check();
+  }
+
+  checkRequestHistory(fastTrack: string) {
+    cy.verifyContentPresence("#sample-test-status", ["Request Submitted"]);
```

Comment:
```suggestion
    cy.verifyContentPresence("#sample-test-status", ["Request Submitted"]);
```

- why are keeping these hardcoded value inside the function, keep it in the test file, make it reusable, checkrequesthistory function will not be having same content all the time